tests/unit/projection-test.js

```diff
@@ -1780,6 +1780,257 @@ module('unit/projection', function(hooks) {
     });
   });
 
+  module('Update projection property with resolved value', function(hooks) {
+    // properties for use for initial state
+    const BOOK_ID = 'isbn:9780439708181';
+    // TODO is this valid? we won't have a real ID yeah?
```

Comment:
not following this TODO; what's the issue?

Reply:
Oh, I copied projection update from a another test module, missed removing the TODO. I will fix that.